src/main/java/com/epam/prd/jjop/helpers/arrays/MismatchAndCompareUnsigned.java

```diff
@@ -0,0 +1,147 @@
+package com.epam.prd.jjop.helpers.arrays;
+
+import java.util.Arrays;
+
+/**+
```

Comment:
```suggestion
/**
```

Unecesssary `+` that breaks javadoc